api-tests/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "api-tests",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Saleor API tests",
+  "main": "N/A",
+  "scripts": {
+    "test": "set -a && . ./.env && set +a && vitest",
```

Comment:
We shouldn't expect our users to be running in a specific shell - `set -a` is UNIX only command

A typical practice is to use `dotenv` which handles loading `.env` files... or not loading `.env` files at all and using tools like [direnv](https://direnv.net/)

It looks like we're already using `dotenv` in vitest config: [api-tests/vitest.config.ts](https://github.com/saleor/saleor/pull/12963/files#diff-fdc5dd4df60190bc0e061ebfa9370a8e968c2da388af1fbbca79a73aa8dc11fc)

Moreover, `.env` file is not present in this PR, should we use `.env` as the default?

Reply:
I've been using direnv in vitest.config.ts, but I changed my approach after @krzysztofzuraw 's suggestion in the original PR: https://github.com/saleor/saleor/pull/12944/files/cea5c85485d6fb2c62886672374a8da63fa0c59d#r1209154106

I've added .env file to .gitignore, but I provided /env-example, so everybody can set up env variables themselves - this way I don't need to hardcode credentials in my code. 

I've also added `  setupFiles: ['dotenv/config'],` to vitest.config.ts in order to use those variables.

If you still want me to make changes, please let me know.

Reply:
I think there are two types of env variables here:
1) used in tests - those can be covered by default vite env [handling](https://vitejs.dev/guide/env-and-mode.html)
2) used to generate GraphQL types based on schema - can't we use the same approach here as in [apps](https://github.com/saleor/saleor-app-template/blob/main/package.json#L10)?

Reply:
Sure - idea there is that you have `schemaVersion` defined in pkg.json like this:
```json
  "saleor": {
    "schemaVersion": "3.10"
  },
```
Reference: https://github.com/saleor/saleor-app-template/blob/main/package.json#L14-L16

and then in `scripts` you can reference this variable as `npm_package_saleor_schemaVersion`